--- ortp-5.3.26/CMakeLists.txt.orig	2024-03-05 20:57:39.456281602 +0100
+++ ortp-5.3.26/CMakeLists.txt	2024-03-05 20:59:43.555609298 +0100
@@ -198,11 +198,11 @@ endif()
 
 
 set(prefix ${CMAKE_INSTALL_PREFIX})
-set(exec_prefix ${prefix}/bin)
-set(libdir ${prefix}/lib)
+set(exec_prefix ${prefix})
+set(libdir ${prefix}/lib${LIB_SUFFIX})
 set(includedir ${prefix}/include)
 set(ORTP_PKGCONFIG_VERSION "${ORTP_VERSION}")
-set(ORTPDEPS_LIBS )
+set(ORTPDEPS_LIBS -lbctoolbox)
 
 configure_file(${PROJECT_SOURCE_DIR}/ortp.pc.in ${PROJECT_BINARY_DIR}/ortp.pc)
 install(FILES ${PROJECT_BINARY_DIR}/ortp.pc DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig")
