--- ortp-0.15.0/src/stun.c~	2009-05-03 14:51:36.000000000 +0200
+++ ortp-0.15.0/src/stun.c	2009-05-03 14:56:07.000000000 +0200
@@ -1109,6 +1109,7 @@
  	fd_set fdSet;
 	int maxFd=0;
 	struct timeval tv;
+	size_t got;
 	int e;
 
         int fd=open("/dev/random",O_RDONLY);
@@ -1132,8 +1133,15 @@
            closesocket(fd);
 	   return random();
 	}
-	read(fd,&tick,sizeof(tick));
+	got = read(fd,&tick,sizeof(tick));
 	closesocket(fd);
+
+	if (got != sizeof(tick))
+	{
+	    ortp_message("stun: Failed to get data from random device\n");
+	    return random();
+	}
+
       }
 #else
 #     error Need some way to seed the random number generator 
