--- ortp-0.16.5/src/stun.c~	2010-11-19 11:32:47.000000000 +0100
+++ ortp-0.16.5/src/stun.c	2011-07-26 13:00:04.172190285 +0200
@@ -1184,7 +1184,7 @@
 		   read(fd,&tick,sizeof(tick));
 		   closesocket(fd);
 	   }
-#elif defined(__GNUC__) && ( defined(__i686__) || defined(__i386__) )
+#elif defined(__GNUC__) && ( defined(__i686__) || defined(__i386__) || defined(__pentium4__) || defined(__athlon__) )
       asm("rdtsc" : "=A" (tick));
 #elif defined(__GNUC__) && defined(__amd64__)
       asm("rdtsc" : "=A" (tick));
