--- ortp-0.16.5/configure.ac~	2011-03-28 20:53:54.000000000 +0200
+++ ortp-0.16.5/configure.ac	2011-07-26 13:00:57.613550509 +0200
@@ -280,8 +280,7 @@
 				
 if test "$ssl_hmac_enabled" = "yes" ; then
 	AC_CHECK_HEADERS(openssl/hmac.h openssl/md5.h)
-	AC_CHECK_LIB(ssl,SSL_CTX_new,[SSL_LIBS="-lssl"])
-	AC_CHECK_LIB(crypto,MD5,[SSL_LIBS="$SSL_LIBS -lcrypto"])
+	AC_CHECK_LIB(crypto,MD5,[SSL_LIBS="-lcrypto"])
 fi
 
 
