--- ortp-0.23.0/src/stun.c.orig	2014-03-01 14:32:05.431008206 +0100
+++ ortp-0.23.0/src/stun.c	2014-03-01 14:33:23.847671582 +0100
@@ -1234,7 +1234,7 @@
 			read(fd,&tick,sizeof(tick));
 			closesocket(fd);
 	   }
-#elif defined(__GNUC__) && ( defined(__i686__) || defined(__i386__) || defined(__amd64__) )
+#elif defined(__GNUC__) && ( defined(__i686__) || defined(__i386__) || defined(__pentium4__) || defined(__athlon__) || defined(__amd64__) )
 	  asm("rdtsc" : "=A" (tick));
 #elif defined (__SUNPRO_CC) && defined( __sparc__ )
 	  tick = gethrtime();
