--- ortp-5.1.72/CMakeLists.txt.orig	2023-04-09 21:59:31.773752891 +0200
+++ ortp-5.1.72/CMakeLists.txt	2023-04-09 22:00:07.306893725 +0200
@@ -203,11 +203,11 @@ endif()
 
 
 set(prefix ${CMAKE_INSTALL_PREFIX})
-set(exec_prefix ${prefix}/bin)
-set(libdir ${prefix}/lib)
+set(exec_prefix ${prefix})
+set(libdir ${prefix}/lib${LIB_SUFFIX})
 set(includedir ${prefix}/include)
 set(ORTP_PKGCONFIG_VERSION "${ORTP_VERSION}")
-set(ORTPDEPS_LIBS )
+set(ORTPDEPS_LIBS -lbctoolbox)
 
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/ortp.pc.in ${CMAKE_CURRENT_BINARY_DIR}/ortp.pc)
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/ortp.pc DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig")
